src/khoj/routers/api_chat.py

```diff
@@ -58,6 +58,19 @@
 
 api_chat = APIRouter()
 
+from khoj.routers.email import feedback
+from pydantic import BaseModel
+
+
+class TextData(BaseModel):
```

Comment:
```suggestion
class FeedbackData(BaseModel):
```

It's better to be more specific in the nomenclature to make it clear what a given class is meant for.

Reply:
good point!